account_reconcile_oca/views/account_move_line.xml

```diff
@@ -0,0 +1,119 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2023 Dixmit
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+
+    <record id="account_move_line_tree_reconcile_view" model="ir.ui.view">
+        <field name="name">account.move.line.tree.reconcile</field>
+        <field name="model">account.move.line</field>
+        <field name="priority">99</field>
+        <field name="arch" type="xml">
+            <tree
+                js_class="reconcile_move_line"
+                create="0"
+                edit="0"
+                export_xlsx="false"
+            >
+                <field name="date" />
+                <field name="move_id" optional="show" />
+                <field name="invoice_due_date" optional="show" />
+                <field name="account_id" optional="show" />
+                <field name="partner_id" />
+                <field name="company_currency_id" invisible="1" />
+                <field name="currency_id" invisible="1" />
```

Comment:
I think it should be `column_invisible`, in version 17 invisible only hides data, not the entire column